src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapSeparatorAndInterfaces.java

```diff
@@ -1,6 +1,6 @@
-package com.puppycrawl.tools.checkstyle.checks.whitespace;
+package com.puppycrawl.tools.checkstyle.checks.whitespace.operatorwrap;
```

Comment:
If we want all files stored in separated packages for every test, we should move  InputSeparatorWrap.java to appropriate package